Dockerfile

```diff
@@ -7,4 +7,4 @@ COPY . .
 RUN pip install .
 
 
-ENTRYPOINT ["pyOTDR"]
+CMD ["pyOTDR"]
```

Comment:
When running the container with the command `make docker-run` pyOTDR was run and `bash` not started. This is because the entrypoint need to explicitely exec the `CMD` (yeah a bit tricky).